### ISSUE TITLE
WowProfile.GetCharacterProfileSummary returns "Error : HTTP/1.1 404 Not Found"

### ISSUE THREAD

**Issue by EricBatlle** [ZeShmoutt/Blizzard-API-For-Unity#2]
Hi, I'm trying to search characters using your API, but It allways returns me a "Error : HTTP/1.1 404 Not Found".

I'm calling the method like this:

```
StartCoroutine(BlizzardAPI.WowProfile.GetCharacterProfileSummary(region: BattleNetRegion.Europe, "zul-jin", "bronto", action_Result: (result) =>
        {
            Debug.LogFormat(result.character_class.ToString());
        }));
```

Values are correct cause I've been trying it from Bnet API website and it works: 

![image](https://user-images.githubusercontent.com/15261880/94007253-7acc6900-fda1-11ea-857f-9c37530beaf8.png)

Is it possible that something is outdated in your API? the "namespace" value seems to do not exist :S

**Comment by EricBatlle** [comment]
Nevermind, my fault, just if somebody else have the same problem...Zul'jin server on the appi should be called like -> zuljin, not zul-jin ^^

**Comment by ZeShmoutt** [comment]
> Nevermind, my fault, just if somebody else have the same problem...Zul'jin server on the appi should be called like -> zuljin, not zul-jin ^^

If you're not sure, you can fetch a list of servers with the Realms Index endpoint and try to find what you're looking for. If you know the realm ID, you can also directly fetch it with the Realm endpoint and you'll get all infos about it, including the slug.

> Is it possible that something is outdated in your API? the "namespace" value seems to do not exist :S

This is because each endpoint only works with a single namespace which is the default one on the API website (e.g. it's always `profile` for Character Profile Summary, `static` for Items, etc.) so it makes no sense to ask the user to input it, it's only adding a risk that you'll write the wrong one and get a 404.

Internally, it uses the generic SendRequest method with the correct namespace for each endpoint :
![image](https://user-images.githubusercontent.com/36408045/94072800-ab8abd80-fdf6-11ea-9b96-43a73727ab60.png)

For the same reason, the Region parameter is only mandatory on endpoints where it makes sense (e.g. characters, realms, etc.), for anything else it's optional and defaults to US, but you can ask for a specific region if you want :
![image](https://user-images.githubusercontent.com/36408045/94073122-3966a880-fdf7-11ea-9a82-e92d9cc96ccb.png)
It doesn't seem to make any difference, though.

TL;DR : I tried to remove as much redundant stuff as I could to make it easier to use.
